pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-discord/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, pkgconfig, pidgin, json_glib }:
+
+stdenv.mkDerivation rec {
+  name = "pidgin-discord-${version}";
+  version = "0.9.2018.04.10.git.9a97886";
```

Comment:
version is supposed to be `unstable-yyyy-mm-dd`: https://nixos.org/nixpkgs/manual/#sec-package-naming